packages/contentful--app-scripts/lib/upload/build-upload-settings.js

```diff
@@ -22,6 +23,21 @@ async function buildAppUploadSettings(options) {
       default: '',
     });
   }
+  if (!skipActivation) {
```

Comment:
We'd need to check for `skipActivation === undefined`, because `skipActivation` is a boolean and we might already have explicitely set it to false.